owl/EASE-STATE.owl

```diff
@@ -27,6 +27,31 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#hasGestallt -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#hasGestallt">
```

Comment:
This might be a good time to rename Gestallt to StateType, and likewise for obj.prop. names.

Meaning, if I don't suggest any other changes, ignore this request; if I do request other changes, might as well put this one along too.

Reply:
true, I remember that this was suggested. However I like the term *Gestallt*. But it's true that meaning of StateType is much more obvious